### PR TITLE
[Localization] Localize arena-tag messages & texts

### PR DESCRIPTION
**Pull request by EnochG1**
## What are the changes?
Add the option to localize text of arena tags.

## Why am I doing these changes?
All text should be localizable.

## What did change?
 - Change hard-coded text for localize > /src/data/arena-tag.ts
 - Added locale files, config, localization keys in each language folders. > config.ts not added yet at draft. (build fails)
 - Translated them to Korean.

### Screenshots/Videos
**Note 1 : I provide just essential test result (to check parameters)
Please comment if you want to check specific case with other languages.**

Note 2 : The list below is developed in the original code, but cannot be tested because these do not occur.
arenaOnRemove
reflectOnAdd
lightScreenOnAdd
auroraVeilOnAdd
conditionalProtectOnAdd
tailwindOnAdd
tailwindOnRemove

<details>
<summary>Click to expand/collapse</summary>

| MessageKey | En | Ko |
| --- | --- | --- |
| arenaOnRemovePlayer | image | image |
| arenaOnRemoveEnemy | image | image |
| mistOnAdd | image | image |
| mistApply | image | image |
| reflectOnAddPlayer | image | image |
| reflectOnAddEnemy | image | image |
| lightScreenOnAddPlayer | image | image |
| lightScreenOnAddEnemy | image | image |
| auroraVeilOnAddPlayer | image | image |
| auroraVeilOnAddEnemy | image | image |
| conditionalProtectOnAddPlayer | image | image |
| conditionalProtectOnAddEnemy | image | image |
| conditionalProtectApply | image | image |
| matBlockOnAdd | image | image |
| wishTagOnAdd | image | image |
| mudSportOnAdd | image | image |
| mudSportOnRemove | image | image |
| waterSportOnAdd | image | image |
| waterSportOnRemove | image | image |
| spikesOnAdd | image | image |
| spikesActivateTrap | image | image |
| toxicSpikesOnAdd | image | image |
| toxicSpikesActivateTrapPoison | image | image |
| [StatusEffect] toxicSpikes | image | image |
| stealthRockOnAdd | image | image |
| stealthRockActivateTrap | image | image |
| stickyWebOnAdd | image | image |
| stickyWebActivateTrap | image | image |
| trickRoomOnAdd | image | image |
| trickRoomOnRemove | image | image |
| gravityOnAdd | image | image |
| gravityOnRemove | image | image |
| tailwindOnAddPlayer | image | image |
| tailwindOnAddEnemy | image | image |
| tailwindOnRemovePlayer | image | image |
| tailwindOnRemoveEnemy | image | image |
| happyHourOnAdd | image | image |
| happyHourOnRemove | image | image |

</details>

## How to test the changes?
Manually. (Using overrides.ts.)

### Note : message key and possible moves matching list

<details>
<summary>Click to expand/collapse</summary>

| MessageKey | Moves |
| --- | --- |
| arenaOnRemove | Moves.MIST Moves.REFLECT Moves.LIGHT_SCREEN Moves.AURORA_VEIL Moves.QUICK_GUARD Moves.WIDE_GUARD Moves.MAT_BLOCK Moves.CRAFTY_SHIELD Moves.SPIKES Moves.TOXIC_SPIKES Moves.STEALTH_ROCK Moves.STICKY_WEB... |
| mistOnAdd | Moves.MIST |
| mistApply | Moves.MIST |
| reflectOnAdd | Moves.REFLECT |
| lightScreenOnAdd | Moves.LIGHT_SCREEN |
| auroraVeilOnAdd | Moves.AURORA_VEIL |
| conditionalProtectOnAdd | Moves.QUICK_GUARD Moves.WIDE_GUARD Moves.MAT_BLOCK Moves.CRAFTY_SHIELD |
| conditionalProtectApply | Moves.QUICK_GUARD Moves.WIDE_GUARD Moves.MAT_BLOCK Moves.CRAFTY_SHIELD |
| matBlockOnAdd | Moves.MAT_BLOCK |
| wishTagOnAdd | Moves.WISH |
| mudSportOnAdd | Moves.MUD_SPORT |
| mudSportOnRemove | Moves.MUD_SPORT |
| waterSportOnAdd | Moves.WATER_SPORT |
| waterSportOnRemove | Moves.WATER_SPORT |
| spikesOnAdd | Moves.SPIKES |
| spikesActivateTrap | Moves.SPIKES |
| toxicSpikesOnAdd | Moves.TOXIC_SPIKES |
| toxicSpikesActivateTrapPoison | Moves.TOXIC_SPIKES |
| [StatusEffect] toxicSpikes | Moves.TOXIC_SPIKES |
| stealthRockOnAdd | Moves.STEALTH_ROCK |
| stealthRockActivateTrap | Moves.STEALTH_ROCK |
| stickyWebOnAdd | Moves.STICKY_WEB |
| stickyWebActivateTrap | Moves.STICKY_WEB |
| trickRoomOnAdd | Moves.TRICK_ROOM |
| trickRoomOnRemove | Moves.TRICK_ROOM |
| gravityOnAdd | Moves.GRAVITY |
| gravityOnRemove | Moves.GRAVITY |
| tailwindOnAdd | Moves.TAILWIND |
| tailwindOnRemove | Moves.TAILWIND |
| happyHourOnAdd | Moves.HAPPY_HOUR |
| happyHourOnRemove | Moves.HAPPY_HOUR |

</details>

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/fr/arena-tag.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "gravityOnAdd": "Gravity intensified!",
+  "gravityOnRemove": "Gravity returned to normal!",
+  "tailwindOnAdd": "The Tailwind blew from behind team!",
+  "tailwindOnAddPlayer": "The Tailwind blew from behind\nyour team!",
+  "tailwindOnAddEnemy": "The Tailwind blew from behind\nthe opposing team!",
+  "tailwindOnRemove": "Team's Tailwind petered out!",
+  "tailwindOnRemovePlayer": "Your team's Tailwind petered out!",
+  "tailwindOnRemoveEnemy": "The opposing team's Tailwind petered out!",
+  "happyHourOnAdd": "Everyone is caught up in the happy atmosphere!",
+  "happyHourOnRemove": "The atmosphere returned to normal.",
```
Comment (Adri1) [inline src/locales/fr/arena-tag.ts:47]:
```suggestion
  "arenaOnRemove": "L’effet de la capacité {{moveName}}\ns’est dissipé !",
  "arenaOnRemovePlayer": "L’effet de la capacité {{moveName}}\ns’est dissipé sur votre équipe !",
  "arenaOnRemoveEnemy": "L’effet de la capacité {{moveName}}\ns’est dissipé sur l’équipe ennemie !",
  "mistOnAdd": "La brume enveloppe l’équipe\de {{pokemonNameWithAffix}} !",
  "mistApply": "La brume empêche les stats de baisser !",
  "reflectOnAdd": "Protection augmente la résistance\naux capacités physiques !",
  "reflectOnAddPlayer": "Protection augmente la résistance\nde l’équipe aux capacités physiques !",
  "reflectOnAddEnemy": "Protection augmente la résistance\nde l’équipe ennemie aux capacités physiques !",
  "lightScreenOnAdd": "Mur Lumière augmente la résistance\naux capacités spéciales !",
  "lightScreenOnAddPlayer": "Mur Lumière augmente la résistance\nde l’équipe aux capacités spéciales !",
  "lightScreenOnAddEnemy": "Mur Lumière augmente la résistance\nde l’équipe ennemie aux capacités spéciales !",
  "auroraVeilOnAdd": "Voile Aurore augmente la résistance\naux capacités physiques et spéciales !",
  "auroraVeilOnAddPlayer": "Voile Aurore augmente la résistance\nde l’équipe aux capacités physiques et spéciales !",
  "auroraVeilOnAddEnemy": "Voile Aurore augmente la résistance\nde l’équipe ennemie aux capacités physiques et spéciales !",
  "conditionalProtectOnAdd": "La capacité {{moveName}}\nprotège l’équipe !",
  "conditionalProtectOnAddPlayer": "La capacité {{moveName}}\nprotège votre équipe !",
  "conditionalProtectOnAddEnemy": "La capacité {{moveName}}\nprotège l’équipe ennemie !",
  "conditionalProtectApply": "{{pokemonNameWithAffix}} est protégé\npar {{moveName}} !",
  "matBlockOnAdd": "{{pokemonNameWithAffix}} se prépare\nà utiliser un tatami pour bloquer les attaques !",
  "wishTagOnAdd": "Le vœu de{{pokemonNameWithAffix}}\nse réalise !",
  "mudSportOnAdd": "La puissance des capacités\nde type Électrik diminue !",
  "mudSportOnRemove": "L’effet de Lance-Boue se dissipe !",
  "waterSportOnAdd": "La puissance des capacités\nde type Feu diminue !",
  "waterSportOnRemove": "L’effet de Tourniquet se dissipe !",
  "spikesOnAdd": "Des {{moveName}} s’éparpillent autour de {{opponentDesc}} !",
  "spikesActivateTrap": "{{pokemonNameWithAffix}} est blessé\npar les picots !",
  "toxicSpikesOnAdd": "Des {{moveName}} s’éparpillent autour de {{opponentDesc}} !",
  "toxicSpikesActivateTrapPoison": "{{pokemonNameWithAffix}} absorbe\n{{moveName}} !",
  "stealthRockOnAdd": "Des pierres pointues lévitent\nautour de {{opponentDesc}} !",
  "stealthRockActivateTrap": "Des pierres pointues\ntranspercent {{pokemonName}} !",
  "stickyWebOnAdd": "Le terrain est couvert d’une {{moveName}}\ndu côté de l’équipe ennemie !",
  "stickyWebActivateTrap": "{{pokemonName}} ennemi\nest pris dans la toile gluante !",
  "trickRoomOnAdd": "{{pokemonNameWithAffix}}\nfausse les dimensions !",
  "trickRoomOnRemove": "Les dimensions faussées\nreviennent à la normale !",
  "gravityOnAdd": "La gravité s’intensifie !",
  "gravityOnRemove": "La gravité est revenue à la normale !",
  "tailwindOnAdd": "Un vent arrière souffle !",
  "tailwindOnAddPlayer": "Un vent arrière souffle\nsur votre équipe !",
  "tailwindOnAddEnemy": "Un vent arrière souffle\nsur l’équipe ennemie !",
  "tailwindOnRemove": "Le vent arrière s’arrête !",
  "tailwindOnRemovePlayer": "Le vent arrière soufflant\nsur votre équipe s’arrête !",
  "tailwindOnRemoveEnemy": "Le vent arrière soufflant\nsur l’équipe ennemie s’arrête !",
  "happyHourOnAdd": "L’ambiance est euphorique !",
  "happyHourOnRemove": "L’ambiance se calme !",
```

**Review by Adri1 — COMMENTED** [review]
Added French

File: src/locales/de/arena-tag.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { SimpleTranslationEntries } from "#app/interfaces/locales";
+
+export const arenaTag: SimpleTranslationEntries = {
+  "arenaOnRemove": "{{moveName}}'s effect wore off.",
```
Comment (CodeTappert) [inline src/locales/de/arena-tag.ts:4]:
```suggestion
  "arenaOnRemove": "Der Effekt von {{moveName}} lässt nach!",
```

File: src/locales/de/arena-tag.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  "gravityOnRemove": "Gravity returned to normal!",
+  "tailwindOnAdd": "The Tailwind blew from behind team!",
+  "tailwindOnAddPlayer": "The Tailwind blew from behind\nyour team!",
+  "tailwindOnAddEnemy": "The Tailwind blew from behind\nthe opposing team!",
+  "tailwindOnRemove": "Team's Tailwind petered out!",
+  "tailwindOnRemovePlayer": "Your team's Tailwind petered out!",
+  "tailwindOnRemoveEnemy": "The opposing team's Tailwind petered out!",
+  "happyHourOnAdd": "Everyone is caught up in the happy atmosphere!",
+  "happyHourOnRemove": "The atmosphere returned to normal.",
+} as const;
```
Comment (CodeTappert) [inline src/locales/de/arena-tag.ts:48]:
```suggestion
export const arenaTag: SimpleTranslationEntries = {
  "yourTeam": "Pokémon auf deiner Seite",
  "opposingTeam": "Pokémon auf gegnerischer Seite",
  "arenaOnRemove": "Der Effekt von {{moveName}} lässt nach!",
  "arenaOnRemovePlayer": "Der Effekt von {{moveName}} lässt auf deiner Seite nach!",
  "arenaOnRemoveEnemy": "Der Effekt von {{moveName}} lässt auf der Seite des Gegners nach!",
  "mistOnAdd": "Pokémon, die auf der Seite von {{pokemonNameWithAffix} kämpfen, werden in Weißnebel gehüllt!!",
  "mistApply": "Der Weißnebel verhindert die Senkung von Statuswerten!",
  "reflectOnAdd": "Reflektor stärkt Pokémon gegen physische Attacken!",
  "reflectOnAddPlayer": "Reflektor stärkt Pokémon auf deiner Seite gegen physische Attacken!",
  "reflectOnAddEnemy": "Reflektor stärkt gegnerische Pokémon gegen physische Attacken!",
  "lightScreenOnAdd": "Lichtschild stärkt Pokémon gegen Spezial-Attacken!",
  "lightScreenOnAddPlayer": "Lichtschild stärkt Pokémon, die auf deiner Seite kämpfen, gegen Spezial-Attacken!",
  "lightScreenOnAddEnemy": "Lichtschild stärkt gegnerische Pokémon gegen Spezial-Attacken!",
  "auroraVeilOnAdd": "Auroraschleier stärkt Pokémon gegen physische und Spezial-Attacken",
  "auroraVeilOnAddPlayer": "Auroraschleier stärkt Pokémon auf deiner Seite gegen physische und Spezial-Attacken!",
  "auroraVeilOnAddEnemy": "Auroraschleier stärkt gegnerische Pokémon gegen physische und Spezial-Attacken!",
  "conditionalProtectOnAdd": "Die Pokémon werden von {{moveName}} behütet!",
  "conditionalProtectOnAddPlayer": "Die Pokémon auf deiner Seite werden von {{moveName}} behütet!",
  "conditionalProtectOnAddEnemy": "Die Pokémon auf der gegnerischen Seite werden von {{moveName}} behütet!",
  "conditionalProtectApply": "{{pokemonNameWithAffix}} wird durch {{moveName}} geschützt!",
  "matBlockOnAdd": "{{pokemonNameWithAffix}} bringt seinen Tatami-Schild in Position!",
  "wishTagOnAdd": "Der Wunschtraum von {{pokemonNameWithAffix}} erfüllt sich!",
  "mudSportOnAdd": "Die Stärke aller Elektro-Attacken wurde reduziert!",
  "mudSportOnRemove": "Lehmsuhler hört auf zu wirken!",
  "waterSportOnAdd": "Die Stärke aller Feuer-Attacken wurde reduziert!",
  "waterSportOnRemove": "Nassmacher hört auf zu wirken!",
  "spikesOnAdd": "Die {{opponentDesc}} sind von Stacheln umgeben!",
  "spikesActivateTrap": "Die {{pokemonNameWithAffix}} wurde durch Stachler verletzt!!",
  "toxicSpikesOnAdd": "Die {{opponentDesc}} sind überall von giftigen Stacheln umgeben",
  "toxicSpikesActivateTrapPoison": "{{pokemonNameWithAffix}} absorbiert die {{moveName}}!",
  "stealthRockOnAdd": "Um die {{opponentDesc}} schweben spitze Steine!",
  "stealthRockActivateTrap": "{{pokemonName}} wird von spitzen Steinen getroffen!!",
  "stickyWebOnAdd": "Am Boden um die {{opponentDesc}} entspinnt sich ein {{moveName}}!",
  "stickyWebActivateTrap": "{{pokemonName}} ist im Klebenetz gefangen!",
  "trickRoomOnAdd": "{{pokemonNameWithAffix}} hat die Dimensionen verdreht!",
  "trickRoomOnRemove": "Die verdrehte Dimension ist wieder normal!",
  "gravityOnAdd": "Die Erdanziehung wurde verstärkt!",
  "gravityOnRemove": "Die Erdanziehung ist wieder normal!",
  "tailwindOnAdd": "Die Pokémon erhalten Rückenwind!",
  "tailwindOnAddPlayer": "Die Pokémon, die auf deiner Seite kämpfen, erhalten Rückenwind!",
  "tailwindOnAddEnemy": "Die gegnerischen Pokémon erhalten Rückenwind!",
  "tailwindOnRemove": "Der Rückenwind hat sich gelegt!",
  "tailwindOnRemovePlayer": "Der Rückenwind auf deiner Seite hat sich gelegt!",
  "tailwindOnRemoveEnemy": "TDer Rückenwind auf gegnerischer Seite hat sich gelegt!",
  "happyHourOnAdd": "Goldene Zeiten sind angebrochen!",
  "happyHourOnRemove": "Die goldenen Zeiten sind vorbei!",
} as const;
```

File: src/locales/de/arena-tag.ts [hunk 4]
```diff
@@ -1,48 +1,50 @@
...
+  "trickRoomOnAdd": "{{pokemonNameWithAffix}} hat die Dimensionen verdreht!",
+  "trickRoomOnRemove": "Die verdrehte Dimension ist wieder normal!",
+  "gravityOnAdd": "Die Erdanziehung wurde verstärkt!",
+  "gravityOnRemove": "Die Erdanziehung ist wieder normal!",
+  "tailwindOnAdd": "Die Pokémon erhalten Rückenwind!",
+  "tailwindOnAddPlayer": "Die Pokémon, die auf deiner Seite kämpfen, erhalten Rückenwind!",
+  "tailwindOnAddEnemy": "Die gegnerischen Pokémon erhalten Rückenwind!",
+  "tailwindOnRemove": "Der Rückenwind hat sich gelegt!",
+  "tailwindOnRemovePlayer": "Der Rückenwind auf deiner Seite hat sich gelegt!",
+  "tailwindOnRemoveEnemy": "TDer Rückenwind auf gegnerischer Seite hat sich gelegt!",
```
Comment (CodeTappert) [inline src/locales/de/arena-tag.ts:47]:
```suggestion
  "tailwindOnRemoveEnemy": "Der Rückenwind auf gegnerischer Seite hat sich gelegt!",
```

File: src/locales/zh_CN/arena-tag.ts [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  "gravityOnAdd": "Gravity intensified!",
+  "gravityOnRemove": "Gravity returned to normal!",
+  "tailwindOnAdd": "The Tailwind blew from behind team!",
+  "tailwindOnAddPlayer": "The Tailwind blew from behind\nyour team!",
+  "tailwindOnAddEnemy": "The Tailwind blew from behind\nthe opposing team!",
+  "tailwindOnRemove": "Team's Tailwind petered out!",
+  "tailwindOnRemovePlayer": "Your team's Tailwind petered out!",
+  "tailwindOnRemoveEnemy": "The opposing team's Tailwind petered out!",
+  "happyHourOnAdd": "Everyone is caught up in the happy atmosphere!",
+  "happyHourOnRemove": "The atmosphere returned to normal.",
```
Comment (40chyan) [inline src/locales/zh_CN/arena-tag.ts:47]:
```suggestion
  "arenaOnRemove": "{{moveName}}的效果消失了！",
  "arenaOnRemovePlayer": "{{moveName}}在我方的效果消失了！",
  "arenaOnRemoveEnemy": "{{moveName}}在敌方的效果消失了！",
  "mistOnAdd": "{{pokemonNameWithAffix}}的一方被\n白雾包围了！",
  "mistApply": "正受到白雾的保护\n能力不会被降低！",
  "reflectOnAdd": "反射壁使\n物理抗性提高了！",
  "reflectOnAddPlayer": "反射壁使我方的\n物理抗性提高了！",
  "reflectOnAddEnemy": "反射壁使敌方的\n物理抗性提高了！",
  "lightScreenOnAdd": "光墙使\n特殊抗性提高了！",
  "lightScreenOnAddPlayer": "光墙使我方的\n特殊抗性提高了！",
  "lightScreenOnAddEnemy": "光墙使敌方的\n特殊抗性提高了！",
  "auroraVeilOnAdd": "极光幕使\n物理和特殊抗性提高了！",
  "auroraVeilOnAddPlayer": "极光幕使我方的\n物理和特殊抗性提高了！",
  "auroraVeilOnAddEnemy": "极光幕使敌方的\n物理和特殊抗性提高了！",
  "conditionalProtectOnAdd": "{{moveName}}\n保护了！",
  "conditionalProtectOnAddPlayer": "{{moveName}}\n保护了我方！",
  "conditionalProtectOnAddEnemy": "{{moveName}}\n保护了敌方！",
  "conditionalProtectApply": "{{moveName}}\n保护了{{pokemonNameWithAffix}}！",
  "matBlockOnAdd": "{{pokemonNameWithAffix}}正在\n伺机使出掀榻榻米！",
  "wishTagOnAdd": "{{pokemonNameWithAffix}}的\n祈愿实现了！",
  "mudSportOnAdd": "电气的威力减弱了！",
  "mudSportOnRemove": "玩泥巴的效果消失了！",
  "waterSportOnAdd": "火焰的威力减弱了！",
  "waterSportOnRemove": "玩水的效果消失了！",
  "spikesOnAdd": "{{opponentDesc}}脚下\n散落着{{moveName}}！",
  "spikesActivateTrap": "{{pokemonNameWithAffix}}\n受到了撒菱的伤害！",
  "toxicSpikesOnAdd": "{{opponentDesc}}脚下\n散落着{{moveName}}！",
  "toxicSpikesActivateTrapPoison": "{{pokemonNameWithAffix}}\n吸收了{{moveName}}！",
  "stealthRockOnAdd": "{{opponentDesc}}周围\n开始浮现出尖锐的岩石！",
  "stealthRockActivateTrap": "尖锐的岩石扎进了\n{pokemonName}}的体内！",
  "stickyWebOnAdd": "对方的脚下\n延伸出了{{moveName}}！",
  "stickyWebActivateTrap": "{{pokemonName}}\n被黏黏网粘住了！",
  "trickRoomOnAdd": "{{pokemonNameWithAffix}}\n扭曲了时空！",
  "trickRoomOnRemove": "扭曲的时空复原了！",
  "gravityOnAdd": "重力变强了！",
  "gravityOnRemove": "重力复原了！",
  "tailwindOnAdd": "从身后\n吹起了顺风！",
  "tailwindOnAddPlayer": "从我方身后\n吹起了顺风！",
  "tailwindOnAddEnemy": "从敌方身后\n吹起了顺风！",
  "tailwindOnRemove": "顺风停止了！",
  "tailwindOnRemovePlayer": "我方的顺风停止了！",
  "tailwindOnRemoveEnemy": "敌方的顺风停止了！",
  "happyHourOnAdd": "大家被欢乐的\n气氛包围了！",
  "happyHourOnRemove": "气氛回复到平常了。",
```

**Review by 40chyan — COMMENTED** [review]
zh_Cn

**Comment by EnochG1** [comment]
FYI, I'm going to continue this work after current beta -> main merge.
I will attach test screenshots after that, please be patient 🙏

File: src/locales/zh_CN/arena-tag.ts [hunk 6]
```diff
@@ -0,0 +1,50 @@
+import { SimpleTranslationEntries } from "#app/interfaces/locales";
+
+export const arenaTag: SimpleTranslationEntries = {
+  "yourTeam": "your team",
+  "opposingTeam": "the opposing team",
```
Comment (40chyan) [inline src/locales/zh_CN/arena-tag.ts:5]:
```suggestion
  "yourTeam": "我方队伍",
  "opposingTeam": "敌方队伍",
```

**Review by 40chyan — COMMENTED** [review]
missing these 2

File: src/locales/fr/arena-tag.ts [hunk 7]
```diff
@@ -0,0 +1,50 @@
+import { SimpleTranslationEntries } from "#app/interfaces/locales";
+
+export const arenaTag: SimpleTranslationEntries = {
+  "yourTeam": "your team",
+  "opposingTeam": "the opposing team",
```
Comment (Adri1) [inline src/locales/fr/arena-tag.ts:5]:
```suggestion
  "yourTeam": "votre équipe",
  "opposingTeam": "l’équipe adverse",
```

**Review by Adri1 — COMMENTED** [review]
Added 2 missing lines in French

**Review by returntoice — APPROVED** [review]
no problem on the Korean translations

**Review by Adri1 — APPROVED** [review]

**Pull request merged**